docs/our_work/ds278_Assurance_Research_Path.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'DS_278: Assurance Research Path'
+summary: 'Investigating new techniques to test and assure AI and machine learning solutions in the NHS'
+origin: 'NHS England'
+tags: ['NLP','LLM','GENAI',Assurance, Machine Learning, Classification, Computer Vision, Work in Progress]
```

Comment:
please check this page for the currently existing tags https://nhsengland.github.io/datascience/our_work/tags/ , and try and align to them, if you dont think theyre appropriate, let me know any suggestions you might have!